OneCommunityGlobal/HighestGoodNetworkApp
# Jiangwei-Apply Tasks Tab time log format to the individual timelogs tabs

# Description
(PRIORITY MEDIUM) Jae: Apply Tasks Tab time log format to the individual timelogs tabs (WIP - Jiangwei)
When using the sort function on the Tasks Tab a manager can see which times are logged to tasks vs projects and if the log was 24, 48, or 72 hours ago → 
I’d like this same format applied to the Time Log page and tabs. 
This is needed so that anyone viewing this information can more easily see when work was completed and what it was logged to (Project or Task)


## Related PRS (if any):
N/A

## Main changes explained:
- Create a 72+ button so Adm/own can check  timelogs tabs before 72h
- Update file B for including new pattern …
- Create file C for introducing new components …
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin/owner user
4. go to dashboard→ timelog→Tasks…

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/56906163/f4924fef-73d5-4a21-a93a-57f0999d592a)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/56906163/32417a93-2560-47f3-8b85-b823680c5d90)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/56906163/99919690-de2e-4993-bf16-3cceb9620cb3)


This page will have an additional '72+' button. When clicked, it will display tasks from others that were set more than 72 hours ago, and the display bar on the left of tasks from more than 72 hours ago should be gray. The background colors for Tangible and Intangible should be different. At the same time, the left bar of the Project/Task should be hollow, and the bar without a Project/Task should be solid.


## Note:
Include the information the reviewers need to know.
